### PR TITLE
[#154 Issue] feat: wallet edit back end name

### PR DESCRIPTION
**Pull request by chedieck**
Description:
Editing a wallet's name also works now.

Test plan:
Go to http://localhost:3000/wallets and try to change the wallets' names. 

Depends on:
- [x] #287

### CODE REVIEW

**Review by johnkuney — CHANGES_REQUESTED** [review]
Sweet, worked well for me. I noticed though if you try using an existing name for instance the api responds with an error, but nothing happens on screen. We should show any error messages on screen

**Comment by chedieck** [comment]
> Sweet, worked well for me. I noticed though if you try using an existing name for instance the api responds with an error, but nothing happens on screen. We should show any error messages on screen

Indeed, we don't support showing errors yet. We definitively should do that, but I think in a next PR, since this problem refers to many other pages besides `/wallets`

**Review by johnkuney — DISMISSED** [review]
Okay cool fair enough.  I think I did actually add some error message handling on the create button form...but probably not done in the best way/shareable with other forms. But yeah fine to tackle in another PR

File: components/Wallet/EditWalletForm.tsx [hunk 1]
```diff
@@ -27,7 +28,13 @@ export default function EditWalletForm ({ wallet, userPaybuttons, refreshWalletL
     if (params.name === '' || params.name === undefined) {
       params.name = wallet.name
     }
-    console.log('edit wallet params', params)
+    void await fetch(`/api/wallet/${wallet.id}`, {
```
Comment (ScottMcDermid) [inline components/Wallet/EditWalletForm.tsx:31]: We should be using the `websiteDomain` environment variable for api requests.

File: components/Wallet/EditWalletForm.tsx [hunk 2]
```diff
@@ -27,7 +28,13 @@ export default function EditWalletForm ({ wallet, userPaybuttons, refreshWalletL
...
+      method: 'PATCH',
```
Comment (ScottMcDermid) [inline components/Wallet/EditWalletForm.tsx:32]: Optionally: `axios` would be more succinct. We're using it in `pages/api/resolvers.index.ts`, at least.
Something like:

```js
void await axios.patch(`${websiteDomain}/api/wallet/${wallet.id}`, params );
```

Should accomplish the same thing I think.

**Review by ScottMcDermid — CHANGES_REQUESTED** [review]

File: components/Wallet/EditWalletForm.tsx (hunk 1 above)
Comment (chedieck) [reply components/Wallet/EditWalletForm.tsx]: Addressed, also made sure this wouldn't be `undefined`.

File: components/Wallet/EditWalletForm.tsx (hunk 2 above)
Comment (chedieck) [reply components/Wallet/EditWalletForm.tsx]: Addressed (check last commit)

**Review by ScottMcDermid — APPROVED** [review]
Looks good!

**Review by Klakurka — APPROVED** [review]

**Pull request merged**